e2e/specs/st_main_menu.spec.ts

```diff
@@ -22,15 +22,22 @@ describe("main menu", () => {
     cy.visit("http://localhost:3000/");
 
     // Make the ribbon decoration line disappear
-    cy.get(".decoration").invoke("css", "display", "none");
+    // cy.get(".decoration").invoke("css", "display", "none");
```

Comment:
Is this meant to be commented out? If so, can we just remove it entirely? (I guess our standard is to remove the ribbon decoration line, but we don't want to do it here for some reason - if that's the case, can you just add a comment to that effect?)

Reply:
Nah. I was having problems with the MainMenu, so I can revert this back. Thanks for catching.